receiver/receivercreator/README.md

```diff
@@ -222,6 +222,21 @@ targeting it will have different variables available.
 | service_type   | The type of the kubernetes service: ClusterIP, NodePort, LoadBalancer, ExternalName   | String                        |
 | cluster_ip     | The cluster IP assigned to the service                                                | String                        |
 
+### Kubernetes Ingress
```

Comment:
Would that be possible to provide an example config as well to illustrate how these can be used in a receiver_creator's config?

Reply:
Sure :)

Reply:
Done @ChrsMark :)